src/frontend/src/pages/WorkPackageForm/CreateWorkPackageCRForm.tsx

```diff
@@ -0,0 +1,29 @@
+import { WbsNumber } from 'shared';
+import WorkPackageForm from './WorkPackageForm';
+import { useCreateSingleWorkPackage } from '../../hooks/work-packages.hooks';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../utils/routes';
+
+interface CreateWorkPackageCrProps {
+  wbsNum: WbsNumber;
+}
+
+const CreateWorkPackageCRForm: React.FC<CreateWorkPackageCrProps> = ({ wbsNum }) => {
+  const history = useHistory();
+
+  if (!wbsNum) throw new Error('WBS number not included in request.');
+
+  const { mutateAsync } = useCreateSingleWorkPackage();
+
```

Comment:
for the payload you'll need to do something a bit weird, you should make a new method `onSubmit = (payload: WorkPackageApiInputs) =>`, you'll need to adjust this type so that crId can be undefined. This method is what you'll pass to mutateAsync below. It is a callback that'll be called when the form is submitted and the input will be the data from the form inputs (in that type I put)

From there inside of the method, you'll need to build a payload to send to the createScopeCR endpoint, you can look at `src\frontend\src\pages\CreateChangeRequestPage\CreateChangeRequest.tsx` for an example of building a payload (for a CR as well)